Gotrue/Client.cs

```diff
@@ -160,29 +72,7 @@ public static Client Instance
 
         private Api api;
```

Comment:
IMO this should be an `IApi` instead of a concrete class

Reply:
Agreed!